joomla-projects/gsoc18_adaptive_images
# Cleaning path and Case Matching of Controller

### Summary 

- Patch for potential errors.

### Testing Instructions

- In media-manager upload any image.
- Go to edit option of that image, goto Smart Crop tab, Select the focus area, save and close the editor.
- Now add that image to the article and see If the image changes when the browser window width is changed.
